payjoin/src/core/mod.rs

```diff
@@ -16,7 +16,13 @@ pub mod send;
 pub use request::*;
 pub(crate) mod into_url;
 pub use into_url::{Error as IntoUrlError, IntoUrl};
+#[cfg(feature = "v2")]
+pub mod time;
 pub mod uri;
+#[cfg(feature = "v1")]
+pub use uri::v1::PjParam as V1PjParam;
+#[cfg(feature = "v2")]
+pub use uri::v2::PjParam as V2PjParam;
```

Comment:
Don't think we want this any longer now that they're exported publicly as `uri::{v1,v2}::PjParam`